high-level-tests/load-testing/config/common.yml

```
@@ -7,3 +7,19 @@ config:
       target: "{{ $processEnvironment.TARGET_DEV_API_URL }}"
     staging:
       target: https://integration.pix.fr
+  plugins:
+    publish-metrics:
+      - type: datadog
+        apiKey: "{{ $processEnvironment.DD_API_KEY }}"
```

Allez ça coûte pas cher
```suggestion
        apiKey: "{{ $processEnvironment.DATADOG_API_KEY }}"
```

---

Merci :+1: